### PR TITLE
Added SlimeVR Wrangler Page, Split "Installing and Connecting" Page, Extra IMU Comparison Info, etc.

### PR DESCRIPTION
**Pull request by NWBx01**
Split the Installing and Connecting page due to its growing scope and somewhat unfocused nature. It has been split into an Initial Setup page and Connecting Trackers page. The Initial Setup page covers server installation, IMU calibration, and confirmation that the SlimeVR driver installed correctly. The Connecting Trackers page covers first time setup using the setup wizard.

- Added page on the usage of SlimeVR Wrangler.
- Added some additional information to the IMU Comparison page.
- For the sake of consistency, page titles and section headings have been capitalized.
- Also for sake of consistency, where appropriate ".html" links have been converted to ".md" in order to preserve local viewability.
- Fixed numbering on the Connecting Trackers page (formerly Installing and Connecting page).
- Converted backslashes to forward slashes
- Fixed SlimeVR Installer image to show all options selected.

### CODE REVIEW

File: tools/slimevr-wrangler.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+layout: page
+nav_order: 2
+parent: Applications and Variants
+---
+
+# SlimeVR Wrangler
+
+This guide will help you install and use SlimeVR Wrangler, which allows the use of Nintendo Switch Joycons as SlimeVR trackers.
+
+![Screenshot of the app running and tracking a single Joy-Con](../assets/img/SlimeVR_Wrangler.png)
```
Comment (carl-anders) [inline tools/slimevr-wrangler.md:11]: Maybe add a comment that this is an old screenshot.

**Comment by calliePepper** [comment]
This is looking solid, the old screenshot comment could be added. There's also updates to some of these pages I think from a recent merge that you might need to bring in

**Comment by NWBx01** [comment]
Rather than leaving a comment about the screenshot being out-of-date, I decided to get an updated image.

**Review by calliePepper — APPROVED** [review]
Looks good!

**Pull request merged**